homework_1/task1/fizzBuzz/fizzBuzz.go

```diff
@@ -0,0 +1,38 @@
+package fizzBuzz
+
+import (
+	"strconv"
+
+	"github.com/pkg/errors"
+)
+
+func PlayFizzBuzz(start int, end int) ([]string, error) {
+
+	if start <= 0 {
+		return nil, errors.New("start should be greater than 0")
+	}
+
+	if end <= 0 {
+		return nil, errors.New("end should be greater than 0")
+	}
+
+	if start > end {
+		return nil, errors.New("start is greater than end")
+	}
+
+	var toPrint []string
```

Comment:
Does this method know what the caller is going to do with the value it returns?
`toPrint` implicates that this variable will be printed but this method has no idea what actually happens (nor it should).

`outputList`, `fizzBuzzOutput` or something similar might be a better name for this variable.